README.rst

```diff
@@ -140,3 +142,16 @@ Database example
     menuBackground=
     status=enabled
 
+Session timeouts
+================
+
+The Server Manager session validity is subjected to forced logoff under the following conditions
+
+* after an idle/inactivity period. It occurs when no user activity is detected in a certain amount of
```

Comment:
Probably the "time" work is missing:
"certain amount of" -> "certain amount of time"